src/compute-client/src/controller/replica.rs

```diff
@@ -150,9 +150,7 @@ where
         }
     }
 
-    /// Instruct orchestrator to ensure replica and run the message loop. In the case of a
-    /// graceful termination after a DropInstance command this method will tell the orchestrator
-    /// to remove the process.
+    /// Instruct orchestrator to ensure replica and run the message loop.
```

Comment:
The message in line 148 above is now moot. Should we inline `run_core` into `run`?

Reply:
I inlined it, it's a bit less verbose that way. The `Ok` branch is now a panic because it cannot happen, unless there is a bug. I would have liked to make the `run_message_loop` function return an `anyhow::Error` directly, instead of a `Result`. But that makes the error handling way more awkward, because you cannot use `?` anymore and have to convert the error and `return` everywhere, so I decided against it.

Maybe we will be able to change that to `Result<!, anyhow::Error>` some day...